.circleci/regenerate.py

```diff
@@ -195,7 +196,7 @@ def generate_upload_workflow(base_workflow_name, os_type, btype, cu_version, *,
     }
 
     if btype == "wheel":
-        d["subfolder"] = "" if os_type == "macos" else cu_version + "/"
+        d["subfolder"] = "" if (os_type == "macos" or os_type == "macos_arm64") else cu_version + "/"
```

Comment:
Nit:

```suggestion
        d["subfolder"] = "" if os_type.startswith("macos") else cu_version + "/"
```

Also applies to the two other places below.